charts/postgres_delete_databases/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Values.namespace }}
+data:
+  db-deletion-script: |-
+    #!/bin/bash
+    echo "DROP DATABASE IF EXISTS $DB_NAME" | psql -d postgres -w
```

Comment:
Maybe consider sanitizing the query input like so
psql -v name="$1" <<< "DROP DATABASE IF EXISTS :'name'"
https://caryrobbins.com/dev/postgres-scripting/

but could also be unnecessary here